### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by saschaishikawa**
Addresses the following issues:
1. change text "Scientist" to "Researcher"
2. parse markdown in classification summary
3. include timestamp in moderation logs

### CODE REVIEW

File: css/moderations.styl [hunk 1]
```diff
@@ -5,6 +5,9 @@ hover-button(color)
     background: lighten(color, 5%)
     border-color: color
 
+.moderations-timestamp
+  font-size 12px
+
```
Comment (chrissnyder) [inline css/moderations.styl]: Not used. Should be dropped (or add the class).

**Comment by chrissnyder** [comment]
Cool, looks good.

For housekeepings sake, make sure the relevant issues are closed (not sure if they'll pick up on the commit message), and add "closes X" to the PR description. Like so: https://github.com/zooniverse/Panoptes-Front-End/pull/1639

**Comment by saschaishikawa** [comment]
@chrissnyder: github pays attention to issue numbers referenced in commit messages, so we're good :)

**Comment by chrissnyder** [comment]
That, but it's also a matter of convention. We've followed the unintentional pattern of putting which issues a PR closes in the PR description. Just make sure that pattern is followed in the future.

**Pull request merged**